sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/schema.yaml

```diff
@@ -0,0 +1,214 @@
+  mode: NULLABLE
+- name: history_removed_today_and_yesterday_users
+  type: INTEGER
+  mode: NULLABLE
+- name: history_search_tap
+  type: INTEGER
+  mode: NULLABLE
+- name: history_search_tap_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_disconnect
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_disconnect_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_login_completed_view
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_login_completed_view_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_login_token_view
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_login_token_view_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_login_view
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_login_view_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_paired
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_paired_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_registration_code_view
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_registration_code_view_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_registration_completed_view
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_registration_completed_view_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_registration_view
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_registration_view_users
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_use_email
+  type: INTEGER
+  mode: NULLABLE
+- name: fxa_use_email_users
+  type: INTEGER
+  mode: NULLABLE
+- name: private_browsing_button_tapped
+  type: INTEGER
+  mode: NULLABLE
+- name: private_browsing_button_tapped_users
+  type: INTEGER
+  mode: NULLABLE
+- name: private_browsing_button_tapped_enter_private_mode
+  type: INTEGER
+  mode: NULLABLE
+- name: private_browsing_button_tapped_enter_private_mode_users
+  type: INTEGER
+  mode: NULLABLE
+- name: private_browsing_icon_tapped
+  type: INTEGER
+  mode: NULLABLE
+- name: private_browsing_icon_tapped_users
+  type: INTEGER
+  mode: NULLABLE
+- name: app_icon_new_private_tab_tapped
+  type: INTEGER
+  mode: NULLABLE
+- name: app_icon_new_private_tab_tapped_users
+  type: INTEGER
+  mode: NULLABLE
+- name: tabs_tray_new_private_tab_tapped
+  type: INTEGER
+  mode: NULLABLE
+- name: tabs_tray_new_private_tab_tapped_users
+  type: INTEGER
+  mode: NULLABLE
+- name: drag_location_bar
+  type: INTEGER
+  mode: NULLABLE
+- name: drag_location_bar_users
+  type: INTEGER
+  mode: NULLABLE
+- name: location_top
+  type: INTEGER
+  mode: NULLABLE
+- name: location_top_users
+  type: INTEGER
+  mode: NULLABLE
+- name: location_bottom
+  type: INTEGER
+  mode: NULLABLE
+- name: location_bottom_users
+  type: INTEGER
+  mode: NULLABLE
+- name: notification_status_authorized
+  type: INTEGER
+  mode: NULLABLE
+- name: notification_status_authorized_users
+  type: INTEGER
+  mode: NULLABLE
+- name: notification_status_notDetermined
+  type: INTEGER
+  mode: NULLABLE
+- name: notification_status_denied
+  type: INTEGER
+  mode: NULLABLE
+- name: notification_alert_setting_notSupported
```

Comment:
Perhaps we should avoid mixing camelCase with snake_case? So in this case, maybe we should consider naming the field notification_alert_setting_not_supported as per our SQL style guide: https://docs.telemetry.mozilla.org/concepts/sql_style.html#variable-names?

(this comment applies to all schema files added by this PR)